libs/labelbox/src/labelbox/data/serialization/ndjson/label.py

```diff
@@ -169,6 +164,7 @@ def _create_non_video_annotations(cls, label: Label):
                     VideoClassificationAnnotation,
                     VideoObjectAnnotation,
                     VideoMaskAnnotation,
+                    RelationshipAnnotation,
```

Comment:
The relationships are currently included twice in the result when calling NDJsonConverter.serialize. 

Because we already generate NDRelationship in `yield from cls._create_relationship_annotations(label)`, there is no need to also do it in `yield from cls._create_non_video_annotations(label)`